### DISCUSSION TITLE
Localization formal or informal

### DISCUSSION THREAD

**Discussion by ezkomoly** [clerk/javascript#3178] [Support request - DO NOT OPEN HERE]
Hi Everyone! 

I would like to contribute with a Hungarian localization. 
In Hungarian, we use different forms of words for formal and informal communication.

For example: 
`'Enter your password'`
Formal would be: 
`'Írja be a jelszavát'`
Informal it would be: 
`'Írd be a jelszavad'`

Which one is the preferred ? Both are used on hungarian websites, depending on the site you are on.

**Comment by jescalan** [comment]
https://github.com/orgs/clerk/discussions/3829
